pkg/testutil/prometheus.go

```diff
@@ -171,6 +184,33 @@ func (p *Prometheus) Start() error {
 	return nil
 }
 
+func (p *Prometheus) WaitPrometheusUp(ctx context.Context) error {
+	if !p.running {
+		return errors.New("method Start was not invoked.")
+	}
+	return runutil.Retry(time.Second, ctx.Done(), func() error {
+		r, err := http.Get(fmt.Sprintf("http://%s/metrics", p.addr))
+		if err != nil {
+			return err
+		}
+
+		if r.StatusCode != 200 {
+			return errors.Errorf("Got non 200 response: %v", r.StatusCode)
+		}
+		return nil
+	})
+}
+
+func (p *Prometheus) Restart() error {
+	if err := p.cmd.Process.Signal(syscall.SIGTERM); err != nil {
+		return errors.Wrap(err, "failed to kill Prometheus. Kill it manually")
+	}
+
+	time.Sleep(time.Second / 2)
```

Comment:
could we use `p.wait()` ? https://golang.org/pkg/os/#Process.Wait
in case of lazy stop running two instances on the same data could cause issues.